Attempt to make code more testable.

adrosenbaum opened the pull request:
All command line commands are built as classes, see
mutacc_auto/commands/
All functions doing anything with the stdout from the system calls are
found in mutacc_auto/parse

adrosenbaum committed:
Attempt to make code more testable.

All command line commands are built as classes, see
mutacc_auto/commands/
All functions doing anything with the stdout from the system calls are
found in mutacc_auto/parse

henrikstranneheim commented on mutacc_auto/parse/parse_scout.py:
```diff
@@ -0,0 +1,39 @@
+
+    for case in cases:
+        case_date = case['updated_at']['$date']
+        case_date = datetime.fromtimestamp(case_date/1000.0)
```
Here it would be nice with get_case_date() def imopoted from scout format

henrikstranneheim replied:
Would be nice with a CONSTANT explaining meaning of 1000

henrikstranneheim commented on mutacc_auto/parse/parse_scout.py:
```diff
@@ -0,0 +1,39 @@
+
+    command = ScoutExportCases()
+
+    cases = get_cases_from_scout(command.check_output(), days_ago = 100)
```
Nice!

henrikstranneheim replied:
If the you use a number which is a constant use a variable constant like CHECK_DAYS_AGO

henrikstranneheim commented on mutacc_auto/parse/parse_housekeeper.py:
```diff
@@ -0,0 +1,21 @@
+
+        tags = fields[-1].split(',')
+
+        if len(tags) == 2 and tags[0].strip() == 'bam':
```
Tags[0].strip() please store in a variable to explain it or add a comment so one knows what it represents

henrikstranneheim reviewed (commented):
Very nicely restructured!

adrosenbaum committed:
making code more modular

Additional restructuring of code, making it more modular.
Commenting of code will be done in short, before PR is merged with
master.

adrosenbaum committed:
Minor changes and commenting of code

adrosenbaum committed:
Some tests added for new code

adrosenbaum committed:
some more tests
